arbitrum-docs/run-arbitrum-node/sequencer/high-availability-sequencer-docs.mdx

```diff
@@ -83,57 +88,80 @@ The requirement is for two types of relays in the architecture:
 
 #### External relays
 
 - Connect to Sequencer Relays (not directly to sequencers)
 - Handle all public feed requests
 - Provide an additional layer of isolation for production sequencers
+- Since these are public facing, ensure they scale appropriately based on your traffic needs:
+
+```console
+helm install external-relay offchainlabs/relay \
+  --set replicaCount=2 \
+  --set configmap.data.chain.id=<your-chain-id> \
+  --set configmap.data.node.feed.input.url=ws://sequencer-relay:9642
+```
 
 You can check [run a feed relay](/run-arbitrum-node/sequencer/run-feed-relay) to see how to set up a relay node.
 
-### 3. Nitro full nodes setup
+### 3. Nitro full node setup
 
-Deploy a set of Nitro full nodes to handle read requests and forward write transactions; please refer to [run a full node](/run-arbitrum-node/03-run-full-node.mdx) for more details.
 
-#### Configuration options:
+```console
+helm install fullnode offchainlabs/nitro \
+  --set replicaCount=2 \
+  --set configmap.data.parent-chain.id=<parent-chain-id> \
+  --set configmap.data.parent-chain.connection.url=<parent-node-url> \
+  --set configmap.data.chain.id=<child-chain-id> \
+  --set configmap.data.execution.forwarding-target=http://sequencer-nitro:8547
+```
 
-**Standard configuration**:
 
-- Forward transactions to a round-robin Kubernetes service containing all sequencer replicas
-- Configure with `--execution.forwarding-target=http://sequencer-nitro.svc:8547`
+#### Send to active configuration (optional)
 
-**Send to active configuration (optional)**:
+To enable Redis-based active sequencer discovery:
 
 - Monitor Redis to identify the active Sequencer
 - Enable with: `-execution.forwarder.redis-url=redis://<redis-url>:6379`
 - Ensure connectivity to individual sequencer services and Redis
 - Test failover scenarios before production deployment
+```console
+helm install fullnode offchainlabs/nitro \
+  --set replicaCount=2 \
+  --set configmap.data.parent-chain.id=<parent-chain-id> \
+  --set configmap.data.parent-chain.connection.url=<parent-node-url> \
+  --set configmap.data.chain.id=<child-chain-id> \
+  --set configmap.data.execution.sequencer.forwarder.redis-url=redis://<redis-url>:6379
+```
 
-**Mutating-only endpoint (optional)**:
+#### Mutating-only endpoint (optional)
 
-- Deploy dedicated nodes for transaction submission only
-- Add this configuration to restrict endpoints to transaction submission only:
+For high availability, it is recommended to route mutating transactions to a fleet of `precheckers` which will then forward them to the sequencer. This can be done by setting up a separate endpoint for mutating transactions, and only allowing calls such as `eth_sendRawTransaction` to be routed to the precheckers which then route to the sequencer. This insulates the sequencer from unnecessary load and allows it to focus on transaction ordering. However, this configuration requires custom load balancing logic and is out of scope for this guide.
```

Comment:
```suggestion
For high availability, it is recommended to route mutating transactions to a fleet of `precheckers`, which will forward them to the sequencer. This can be done by setting up a separate endpoint for mutating transactions and only allowing calls such as `eth_sendRawTransaction` to be routed to the precheckers, which then route to the sequencer. This insulates the sequencer from unnecessary load and enables it to focus on transaction ordering. However, this configuration requires custom load balancing logic and is out of scope for this guide.
```